twentytwenty-blocks/experimental-theme.json

```diff
@@ -0,0 +1,13 @@
+{
+	"color": {
+		"primary": "#cd2653",
+		"background": "#f5efe0",
+		"text": "#000"
+	},
+	"typography": {
```

Comment:
As per feedback, the variable generation mechanism is not going to make it into https://github.com/WordPress/gutenberg/pull/20530 so I have to change the typography variables to something like:

```
	"typography": {
		"font-base": 16,
		"font-scale": 1.2,
		"font-weight-base": 400,
		"font-weight-heading": 400,
		"line-height-base": 1.5,
		"line-height-heading": 1.5
	}
```

Reply:
Note how we declare base and heading alternatives for weight & line-height.

I reckon this is in flux and it puts so it takes a bit of following up to keep up with the changes. I'm happy to submit these as they happen in the Gutenberg PR if that's helpful.

Reply:
Note for font-sizes: the current UI proposal by @karmatosed https://github.com/WordPress/gutenberg/issues/21030 only exposes base and scale in the UI.

Hopefully, in a subsequent iteration, we're able to use `calc` or something along those lines in the `theme.json` itself. However, for this PR we're just pushing computation to CSS-land, based on `font-base` and `font-scale` values. See [example](https://github.com/WordPress/gutenberg/pull/20530/files?file-filters%5B%5D=.scss#diff-f2f5efb13e0e16e61312aa9b0c028ae1R17).

Reply:
I'm pushing at https://github.com/nosolosw/theme-experiments/tree/add/twentytwenty-globalstyles what I have to make this work with the latest version of the Gutenberg PR.